content/english/python-interactive/activity-3.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Activity 3 - Welcome Customer to your Restaurant & Display Menu"
+date: 2019-07-25T13:24:17-07:00
+weight: 3
+draft: false
+---
+## Welcome Customer to your Restaurant & Display Menu
+Now that we have both a restaurant banner to display to the customer and a menu stored in your Python Program, let's display your menu to the customer and welcome them in. In this activity, we're going to write 2 helper functions to accomplish this.
+{{% notice %}}
+## Helper Functions
+Helper functions are functions that get called in a main function that runs all of the code for this program. Each of these helper functions works toward one simple goal.
+{{% /notice %}}
+
+The first helper function we'll be writing is `printMenu()`, a function that will iterate through the `dictMenu` dictionary and result in a Menu that looks like the example shown below. Since dictMenu's values are all lists with the price of the item as the first element and the description as the second element, we want to make sure that when we print out each item's information, we access each element of that list by using indexing.
```

Comment:
```suggestion
The first helper function we'll be writing is `printMenu()`, a function that will iterate through the `dictMenu` dictionary and result in a Menu that looks like the example shown below. Since` dictMenu`'s values are all lists with the price of the item as the first element and the description as the second element, we want to make sure that when we print out each item's information, we access each element of that list by using indexing.
```